test/cmd/quota.sh

```diff
@@ -13,7 +13,7 @@ os::test::junit::declare_suite_start "cmd/quota"
 
 os::test::junit::declare_suite_start "cmd/quota/clusterquota"
 
-os::cmd::expect_success 'oc new-project quota-foo --as=deads'
+os::cmd::expect_success 'oc new-project quota-foo --as=deads --as-group=system:authenticated --as-group=system:authenticated:oauth'
```

Comment:
this is probably the most painful part of the switch.  We have a non-standard group applied to our oauth users and we use it only for project requests.